db/migrate/20230324164939_add_constraints_to_product_details.rb

```diff
@@ -0,0 +1,6 @@
+class AddConstraintsToProductDetails < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
Why are you writing another migration for adding the constraints at the DB level when you can just edit the previous one.

Reply:
Because the best practice is to add new migration rather than editing the previous one.

Reply:
It's for those cases whose migration has already run and there's no way to down that migration but in your case, you can down the migrations and add whatever you want and make it up again.